### ISSUE TITLE
Showing a single section at a time

### ISSUE THREAD

**Issue by ldesousa** [ryangrose/easy-pandoc-templates#7]
Not a bug, feature request at best. Do you think it is possible to modify one of the templates into a structure only showing a section at a time? Every time an item would be clicked in the TOC the current section would hide and another one would show. The end result would be something similar to what mdBook or Bookdown create.

I understand `div` elements can be hidden/shown with javascript. Perhaps creating a div per section could achieve this, but it requires a loop over the `pandoc` sections content. Is that even possible? 

Thank you for reading.

**Comment by ryangrose** [comment]
Yeah that sounds like a great idea. I imagine this would be easiest with a new template
